lighthouse-core/audits/metrics.js

```diff
@@ -115,7 +120,7 @@ class Metrics extends Audit {
     const details = {
       type: 'debugdata',
       // TODO: Consider not nesting metrics under `items`.
-      items: [metrics],
+      items: [metrics, {lcpInvalidated: traceOfTab.lcpInvalidated}],
```

Comment:
maybe this should be `observedLcpInvalidated` or something but I can't imagine it ever mattering (and maybe like `fmpFellBack` it'll eventually just go unused in the artifacts)